platform/cavium/cavm-sai.mk

```diff
@@ -1,12 +1,20 @@
 # Cavium SAI
 
+CAVM_SAI_GITHUB = https://github.com/garyachy/OpenXPS/raw/sai_0.9.5/SAI/cavm-sai/
```

Comment:
Are these files going to be updated within same branch?
If yes, use commit hash in URL instead of branch name.